ReactiveCocoa/Swift/Property.swift

```diff
@@ -501,13 +504,15 @@ public final class Property<Value>: PropertyProtocol {
 				fatalError("Receive unexpected error from a producer of `NoError` type: \(error)")
 			}
 		}
+
+		// Verify that an initial is sent. This is friendlier than deadlocking
+		// in the event that one isn't.
 		guard value != nil else {
```

Comment:
Since you're accessing this later you should do `guard let value = value` here to make line 515 safe at compile time.